lakey_service/account/templates/get_token.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Get Lakey Token</title>
+        <meta
+            name="google-signin-client_id"
+            content="{{ client_id }}">
+
+        <link
+            href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans"
+            rel="stylesheet">
+        <style>
+            body {
+                background: #e4e4e4;
+            }
+
+            #main {
+                height: 200px;
+                width: 100%;
+                font-family: 'IBM Plex Sans', sans-serif;
+            }
+
+            h2 {
+                font-weight: normal;
+            }
+
+            #authenticate, #done, #wait {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                justify-content: center;
+            }
+
+            #done, #wait {
+                display: none;
+            }
+        </style>
+        <script
+            type="text/javascript">
+
+
+            function renderButton() {
+                gapi.signin2.render('google-auth-button', {
+                    'scope': 'profile email',
+                    'width': 240,
+                    'height': 50,
+                    'longtitle': true,
+                    'theme': 'dark',
+                    'onsuccess': onSuccess,
+                    'onfailure': onFailure
+              });
+            }
+
+            function onSuccess(googleUser) {
+                let $done = document.getElementById('done');
+                let $wait = document.getElementById('wait');
+                let $authenticate = document.getElementById('authenticate');
+
+                let profile = googleUser.getBasicProfile();
+                if(googleUser.getHostedDomain() != 'viessmann.com') {
```

Comment:
let's not check it here any more since we already check it at the command level

Reply:
was considering this, honestly...
left it also here in order to leave some message on UI level for the user.
But if it makes you uncomfortable too... it's gone! ;)